tasks/main.yml

```
@@ -1,19 +1,18 @@
 ---
-- name: Install MariaDB on RedHat Systems
-  when: ansible_os_family == 'RedHat'
-  ansible.builtin.dnf:
-    name:
-      - mariadb-server
-      - mariadb
-      - python3-PyMySQL
 
-- name: Install MariaDB on Debian Systems
-  when: ansible_os_family == 'Debian'
-  ansible.builtin.apt:
-    name:
-      - mariadb-server
-      - mariadb-client
-      - python3-pymysql
+- name: Loading platform variables
+  include_vars: "{{ item }}"
+  with_first_found:
+    - "{{ ansible_distribution }}-{{ ansible_distribution_release }}.yml"
+    - "{{ ansible_distribution }}.yml"
+    - "{{ ansible_os_family }}.yml"
+  tags: always
```

Why this complicated? We know that only the last will ever catch, won't it?

```suggestion
  include_vars: "{{ ansible_os_family }}.yml"
  tags: always
```

---

agreed. I had put the others there as a preparation for possible further distribution deviations, but there seem to be no more